jep/8/README.adoc

```diff
@@ -144,16 +141,14 @@ and then seek reimbursement for the expenses. Pre-approval is still required in
 
 === Liability
 
-Jenkins project has no legal entity which can commit
-on expenses.
+Jenkins project has no legal entity which can commit on expenses.
 
 This means:
 
 * Approval by GSoC org admins does not guarantee the payments
-* Jenkins project, GSoC org admins or SPI cannot guarantee ETA of any payment or, in the worst case,
+* Jenkins project, GSoC org admins or SPI cannot guarantee the timing of any payment or, in the worst case,
 the payment itself
-* Jenkins Project or SPI cannot issue guarantee letters for
-visa applications or other or other travel documents.
+* Jenkins Project or SPI cannot issue guarantee letters for visa applications or other or other travel documents.
```

Comment:
"Or other" is duplicated (not a new issu tho)

Reply:
```suggestion
* Jenkins Project or SPI cannot issue guarantee letters for visa applications or other travel documents.
```